.github/workflows/security.yml

```diff
@@ -0,0 +1,47 @@
+name: Security Scan
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - renovate/**
+  pull_request:
+  schedule:
+    - cron: "0 3 * * *"
+
+jobs:
+  trivy:
+    name: Trivy security scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
I'd update it to`checkout@v3`

Reply:
Updated 9a9f97ed4d344dc8fe98f60bf8e947edf9794dc2, thanks!